admin/metabox/class-metabox.php

```diff
@@ -718,10 +718,25 @@ public function save_postdata( $post_id ) {
 			}
 			else {
 				if ( isset( $_POST[ $field_name ] ) ) {
-					$data = WPSEO_Utils::sanitize_text_field( wp_unslash( $_POST[ $field_name ] ) );
+					$data = wp_unslash( $_POST[ $field_name ] );
+
+					// For multi-select
+					if ( is_array( $data ) ) {
+						$data = array_map( array( 'WPSEO_Utils', 'sanitize_text_field' ), $data );
```

Comment:
I think we might consider moving this part to the utils. But it's not something we should do now.